README.md

```diff
@@ -1,12 +1,14 @@
+
 <a href="https://istio.io/">
-    <img src="logo/istio-bluelogo-whitebackground-unframed.svg" alt="Istio logo" title="Istio" align="right" width="100" />
+    <img src="https://github.com/istio/istio/raw/master/logo/istio-bluelogo-whitebackground-unframed.svg"
+         alt="Istio logo" title="Istio" height="100" width="100" />
```

Comment:
@shamsher31 The smaller size looks much better, but the arrangement still looks off. Can you try ordering the top items just like K8s does? 1. `# Istio` heading, 2. Go report and go doc, 3. Icon.

Reply:
Done, now it looks better.